lms/templates/shoppingcart/registration_code_redemption.html

```diff
@@ -51,8 +52,8 @@ <h1 class="title">
                 "to see if you're enrolled in the course, or contact your "
                 "company's administrator."
               ).format(
-                link_start=u'<a href="{url}">'.format(url=reverse('dashboard')),
-                link_end='</a>',
+                link_start=HTML(u'<a href="{url}">'.format(url=reverse('dashboard'))),
```

Comment:
Since you're now passing in ``HTML`` does this need to be wrapped in ``Text`` on the outside?